styles.css

```diff
@@ -349,4 +552,82 @@ input[type=submit]:hover {
   display: flex;
   position: relative;
   padding: 30px 0;
+  justify-content: center;
+}
+
+@media screen and (min-width: 800px) {
+  .contact_section {
+    background-image: url(/images/image-geometry_6.svg), url(/images/image-geometry_7.svg);
+    background-repeat: no-repeat;
+    background-position: top left, bottom right;
+    background-size: 16% 23%;
+  }
+
+  .text_area {
+    width: 81%;
+    padding: 180px;
+    padding-bottom: 0%;
+  }
+
+  .contact_section h2 {
+    font-size: 110px;
+  }
+}
+
+footer {
+  bottom: 0;
+  left: 0;
+  height: 150px;
+  position: fixed;
+  width: 100%;
+  background-color: #1a2236;
+  padding-top: 40px;
+}
```

Comment:
Well done for adding the footer neatly. Kindly stick to the design as much as possible. The figma design does not seem to specify it to have a fixed position.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206877394-128e0076-bed0-4dff-8cf4-8415ce012d01.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206877416-82ef3495-4af4-4d86-babb-84af94a39655.png)